src/index.html

```diff
@@ -11,6 +11,43 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="search search-large" data-qa="big">
+      <label class="rectangle">
+        <label class="icon-box large-icon-box">
+          <img
+          class="large-search-icon"
+          src="./images/Search.svg"
+          alt="search"
+          >
+        </label>
+        <input type="search"
+        id="large-search"
+        name="largeSearch"
+        class="text text-normal"
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress"
+        >
+      </label>
+    </form>
+
+    <form class="search search-small" data-qa="small">
+      <label class="rectangle">
+        <label class="icon-box small-icon-box">
+          <img
+          class="small-search-icon"
+          src="./images/Search.svg"
+          alt="search"
+          >
+        </label>
+        <input type="search"
+        id="small-search"
+        name="smallSearch"
+        class="text text-small"
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress"
+        >
+      </label>
+    </form>
+
```

Comment:
```suggestion
    <form class="search" data-qa="big">
      <label class="label">
          <img
            class="search-icon"
            src="./images/Search.svg"
            alt="search"
          >
        </label>
        <input type="search"
          id="large-search"
          name="largeSearch"
          class="input"
          placeholder="Try &quot;Los Angeles&quot;"
          data-qa="keypress"
        >
      </label>
    </form>

    <form class="search search-small" data-qa="small">
      <label class="label">
          <img
            class="search-icon search-icon-small"
            src="./images/Search.svg"
            alt="search"
          >

        <input 
          type="search"
          id="small-search"
          name="smallSearch"
          class="input input-small"
          placeholder="Try &quot;Los Angeles&quot;"
          data-qa="keypress"
        >
      </label>
    </form>

```
lets make it work like this